test/e2e_node/util.go

```diff
@@ -305,7 +305,7 @@ func logKubeletLatencyMetrics(ctx context.Context, metricNames ...string) {
 	for _, key := range metricNames {
 		metricSet.Insert(kubeletmetrics.KubeletSubsystem + "_" + key)
 	}
-	metric, err := e2emetrics.GrabKubeletMetricsWithoutProxy(ctx, fmt.Sprintf("%s:%d", nodeNameOrIP(), ports.KubeletReadOnlyPort), "/metrics")
+	metric, err := e2emetrics.GrabKubeletMetricsWithoutProxy(ctx, fmt.Sprintf("%s:%d", nodeNameOrIP(ctx), ports.KubeletReadOnlyPort), "/metrics")
```

Comment:
```suggestion
        host := net.JoinHostPort(nodeNameOrIP(ctx), strconv.Itoa(ports.KubeletReadOnlyPort))
	metric, err := e2emetrics.GrabKubeletMetricsWithoutProxy(ctx, host, "/metrics")
```

Reply:
good idea to use `net.JoinHostPort` and `ports.KubeletReadOnlyPort` in tests above as well